repair/repair.cc

```diff
@@ -1311,6 +1377,31 @@ static future<> repair_ranges(lw_shared_ptr<repair_info> ri) {
     });
 }
 
+static future<> try_wait_for_hints_to_be_replayed(repair_uniq_id id, std::vector<gms::inet_address> source_nodes, std::vector<gms::inet_address> target_nodes) {
+    auto& sp = service::get_local_storage_proxy();
+    rlogger.info("repair id {}: started replaying hints before repair, target nodes: {}", id, target_nodes);
```

Comment:
can we log both src and target nodes here

Reply:
Done